micrometer-core/src/main/java/io/micrometer/core/instrument/logging/LoggingMeterRegistry.java

```diff
@@ -126,22 +128,24 @@ protected void publish() {
                     double count = counter.count();
                     if (!config.logInactive() && count == 0)
                         return;
-                    loggingSink.accept(print.id() + " throughput=" + print.rate(count));
+                    loggingSink
+                        .accept(print.id() + " delta_count=" + ((int) count) + " throughput=" + print.rate(count));
```

Comment:
It was indeed for formatting reasons and I had not taken into account that counters could be increased with decimal values.\
I've found function wholeOrDecimal to format delta counts, I'll use it to avoid the cast.